src/components/__tests__/TimePicker.test.tsx

```diff
@@ -0,0 +1,150 @@
+import TimePicker from '../TimePicker'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+describe('TimePicker', () => {
+    const timePickerTestId = 'time-picker'
+
+    describe('no min/max entered', () => {
+        it('displays the correct time ranges with default time span', () => {
+            render(<TimePicker />)
+
+            const select = screen.getByTestId(timePickerTestId) as HTMLSelectElement
+
+            expect(select).toBeInTheDocument()
+
```

Comment:
```suggestion
```

Not needed. By querying with any `screen.get*` function, it already guarantees that what you queried for was found.